config/_default/menus/menus.en.yaml

```diff
@@ -1013,6 +1013,11 @@ main:
     parent: infrastructure_livecontainers
     identifier: infrastructure_livecontainers_configuration
     weight: 301
+  - name: Container Images View
+    url: infrastructure/containers/container_images
+    parent: infrastructure_livecontainers
+    identifier: infrastructure_livecontainers_container_images
+    weight: 302
```

Comment:
Alicia, can you also change the weights for the menu items below 
<img width="752" alt="image" src="https://github.com/DataDog/documentation/assets/41168354/59485dc4-cad6-466d-9e51-070c47220294">